app.gummi.gummi.yml

```diff
@@ -0,0 +1,90 @@
+id: app.gummi.gummi
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gummi
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.texlive
+
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    directory: texlive
+    subdirectories: true
+    autodelete: true
+    version: '22.08'
+
+finish-args:
+  - --share=ipc
+  - --filesystem=xdg-documents
+
+modules:
+  - shared-modules/intltool/intltool-0.51.json
+  
+  - name: mkdir-texlive
+    buildsystem: simple
+    build-commands:
+      - install -d /app/texlive
+  
+  - name: gtksourceview3
+    config-opts:
+      - --disable-gtk-doc
+      - --enable-maintainer-mode
+      - --enable-gtk-doc=no
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz
+        sha256: 691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd
+      - type: shell
+        commands:
+          - cp -p /usr/share/automake-*/config.{sub,guess} .;
+          - autoreconf -vfi;
+    post-install:
+      - install -p -D -m 0644 "COPYING" -t "${FLATPAK_DEST}/share/licenses/gtksourceview/";
+
+  - name: poppler
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DCMAKE_INSTALL_LIBDIR=/app/lib"
+      - "-DCMAKE_INSTALL_INCLUDEDIR=/app/include"
+      - "-DBUILD_GTK_TESTS=OFF"
+      - "-DBUILD_CPP_TESTS=OFF"
+      - "-DENABLE_CPP=OFF"
+      - "-DENABLE_BOOST=OFF"
+      - "-DENABLE_GOBJECT_INTROSPECTION=OFF"
+      - "-DENABLE_LIBOPENJPEG=openjpeg2"
+      - "-DENABLE_QT5=OFF"
+      - "-DENABLE_QT6=OFF"
+      - "-DCMAKE_BUILD_TYPE=Release"
+    cleanup:
+      - "/bin"
+    sources:
+      - type: archive
+        url: https://poppler.freedesktop.org/poppler-23.01.0.tar.xz
+        sha256: fae9b88d3d5033117d38477b79220cfd0d8e252c278ec870ab1832501741fd94
+        x-checker-data:
+          is-important: true
+          type: anitya
+          project-id: 3686
+          url-template: https://poppler.freedesktop.org/poppler-$version.tar.xz
+
+  - name: gtkspell3
+    config-opts:
+      - --disable-gtk-doc
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/gtkspell/gtkspell3-3.0.10.tar.xz
+        sha256: b040f63836b347eb344f5542443dc254621805072f7141d49c067ecb5a375732
+    post-install:
+      - install -p -D -m 0644 "COPYING" -t "${FLATPAK_DEST}/share/licenses/gtkspell/";
+
+  - name: texlive
+    buildsystem: simple
+    build-commands:
+      - sh /usr/lib/sdk/texlive/install.sh
```

Comment:
why no having this done with the install on line 27?

Reply:
see also the comment about using the extension. it's either or.

Reply:
Removed this module at https://github.com/flathub/flathub/pull/3789/commits/b6aaf483bf3d9840eadb6d6a189517a92d2dab87